homeassistant/components/http.py

```diff
@@ -131,13 +131,13 @@ def start_wsgi_server(event):
         """Start the WSGI server."""
         server.start()
 
-    hass.bus.listen_once(EVENT_HOMEASSISTANT_START, start_wsgi_server)
+    hass.bus.async_listen_once(EVENT_HOMEASSISTANT_START, start_wsgi_server)
```

Comment:
If I use sync version, get freeze in this method

    (hass) Michaels-MacBook:hass michaelkuty$ ./hass-slave.py 
    INFO:requests.packages.urllib3.connectionpool:Starting new HTTP connection (1): localhost
    DEBUG:requests.packages.urllib3.connectionpool:"GET /api/ HTTP/1.1" 200 27
    DEBUG:asyncio:Using selector: KqueueSelector
    INFO:requests.packages.urllib3.connectionpool:Starting new HTTP connection (1): localhost
    DEBUG:requests.packages.urllib3.connectionpool:"GET /api/states HTTP/1.1" 200 2
    INFO:homeassistant.loader:Loaded api from homeassistant.components.api
    INFO:homeassistant.loader:Loaded http from homeassistant.components.http